compose-builder/add-security.yml

```diff
@@ -228,19 +246,6 @@ services:
 # end of containers for reverse proxy
 
   database:
-    env_file:
-      - database-security.env
-    command: |
-      /bin/sh -c "
-      until [ -r $${REDIS5_PASSWORD_PATHNAME} ] && [ -s $${REDIS5_PASSWORD_PATHNAME} ]; do sleep 1; done
-      exec /usr/local/bin/docker-entrypoint.sh --requirepass `cat $${REDIS5_PASSWORD_PATHNAME}` \
-        --dir /data \
-        --save 900 1 \
-        --save 300 10 \
-        --save 60 10000
-      "
-    volumes:
-      - /tmp/edgex/secrets/edgex-redis:/tmp/edgex/secrets/edgex-redis:z
     depends_on:
       - vault-worker
```

Comment:
Do we need this `database` section at all. Only thin left is the depends_on which doesn't seem to be needed.

Reply:
Agree with lenny.  redis-bootstrap needs to depend on database and vault-worker.  This can go.

Reply:
Done